src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -332,13 +333,13 @@ protected bool IsAttributeConditionallyOmitted(NamedTypeSymbol attributeType, Sy
 
                         // Named argument
                         // TODO: use fully qualified identifier name for boundNamedArgumentsSet
-                        string argumentName = argument.NameEquals.Name.Identifier.ValueText;
+                        string argumentName = argument.NameEquals.Name.Identifier.ValueText!;
                         if (boundNamedArgumentsBuilder == null)
```

Comment:
>boundNamedArgumentsBuilder == null [](start = 28, length = 34)

consider changing this test to check `boundNamedArgumentsSet` instead, since the two locals are always initialized together (or not), then the suppression can be removed.

Reply:
Our approach for nullable has been to not change any semantic behavior, just to annotate what we are currently doing. I'm going to keep this going forward.

---
In reply to: [367181664](https://github.com/dotnet/roslyn/pull/40946#discussion_r367181664) [](ancestors = 367181664)